packages/utils/lib/environment/parse.ts

```diff
@@ -123,6 +125,19 @@ export const ENVS = z.object({
         .regex(/^[a-zA-Z0-9_-]+$/)
         .optional()
         .default('nango_runners'),
+    RUNNER_LAMBDA_FLEET_ID: z.string().optional().default('nango_runners_lambda'),
+    RUNNER_FLEET_RULES: z
```

Comment:
Rules are provided as an environment variable. This should be a JSON array of [json-rules-engine](https://github.com/cachecontrol/json-rules-engine) rules.